src/pudl/transform/eia861.py

```diff
@@ -805,9 +828,14 @@ def transform(raw_dfs, eia861_tables=pc.pudl_tables["eia861"]):
         "balancing_authority_eia861": balancing_authority,
         "sales_eia861": sales,
         "demand_response_eia861": demand_response,
+        "distribution_systems_eia861": distribution_systems,
     }
     tfr_dfs = {}
 
+    # run early transform to change year to date and fix eia na cols.
+    # for table_name, table in raw_dfs.items():
+    #     raw_dfs[table_name] = _early_transform(table)
+
```

Comment:
Please go ahead and delete things once the new placement works.

Reply:
will do